src/Mvc/Mvc.ViewFeatures/src/Infrastructure/HttpUriHelper.cs

```diff
@@ -3,57 +3,17 @@
 
 using System;
 using Microsoft.AspNetCore.Components.Services;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Extensions;
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures
 {
     internal class HttpUriHelper : UriHelperBase
```

Comment:
Out of interest, why is it called `HttpUriHelper`? Is this specific to prerendering, and if so, could it be called `PrerenderingUriHelper` instead?

Reply:
It was called that way because it was initialized with the HttpContext. I plan to separate the terminology even more between the basic rendering MVC does and the prerendering. My current thinking is to name the basic MVC Rendering with Static and the full thing Prerendering